src/lib/utils/logger/Logger.ts

```diff
@@ -38,7 +38,7 @@ export class Logger implements ILogger {
 	public write(level: LogLevel, ...values: readonly unknown[]): void {
 		if (!this.has(level)) return;
 		const method = Logger.levels.get(level);
-		if (typeof method === 'string') console[method](...values);
+		if (typeof method === 'string') console[method](`[${method.toUpperCase()}]`, ...values);
```

Comment:
Make this an option that defaults to true so people can still disable it with `clientOptions.logger.methodUppercase: false`

Reply:
Does it really matter? It's the preincluded logger, why does it matter if it logs `[info]` or `[INFO]`

Reply:
You know how developers are. Eventually, we'll have someone who complains about it and not adding the option means we'll have to tell them "you can make your own logger plugin" which isn't going to go over well I'm sure. Better to play it safe and just add the option, just in case.

Reply:
Also, if it becomes an option in the built-in logger, extensions would be forced to implement it as well, and the plugin has colours and fine customisation that the built-in one lacks of.

Reply:
I'll pass, I don't see any benefit im bloating the logger options for something as small as this. I only added this mainly for myself so I know what's logging from the registries, there's no reason to open Pandora's Box

Reply:
```
export interface ClientLoggerOptions {
	level?: LogLevel;
	methodUppercase?: boolean;
	instance?: ILogger;
}
```

wdym forced to implement? Just make it an optional property...